src/components/auth0-login-zone/auth0-login-zone.js

```diff
@@ -16,8 +16,8 @@ export const Auth0LoginZone = ({ auth: { isAuthenticated, isLoading, loginWithRe
     <login-zone>
       {
         isAuthenticated
-          ? <button onClick={() => { logout({ returnTo }) }}><LoginSvgsAndText text="LOG OUT" /></button>
-          : <button onClick={ loginWithRedirect }><LoginSvgsAndText text="LOG IN" /></button>
+          ? <button data-testid="login-button" onClick={() => { logout({ returnTo }) }}><LoginSvgsAndText text="LOG OUT" /></button>
+          : <button data-testid="login-button" onClick={ loginWithRedirect }><LoginSvgsAndText text="LOG IN" /></button>
```

Comment:
these two buttons have the same `data-testid`

Reply:
yeah but they are part of a ternary expression so they aren't going to show up at the same time so they are unique.

Reply:
Latest version of the code should have these changes.